integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24Tests.kt

```diff
@@ -327,21 +328,35 @@ private const val expectedSep24Info =
         "enabled": true
       },
       "USD": {
-        "enabled": true
+        "enabled": true,
```

Comment:
We are not implementing the `min_amount` and `max_amount` in AP and we actually decided to move the `fee` model to the SEP-38 endpoint. We should not add these fields back.

Reply:
Not sure I understand, we use these fields for request validation. https://github.com/stellar/java-stellar-anchor-sdk/blob/a126540141af5e32de03ca00f5ef70a831b26988/core/src/main/java/org/stellar/anchor/sep24/Sep24Service.java#L124-L131

Reply:
These are legacy. We should remove these fields from the code.